src/Car.java

```diff
@@ -50,7 +60,20 @@ public void accelerate(int amount){
 	 * @return string representation of object. 
 	 */
 	public String toString(){
-		// not implemented!
-		return("");
+		
+		return("Color: " + color + ", Speed: " + speed);
+		
+	}
+
+	public void decelerate(int amount) {
```

Comment:
I think you should rethink this logic for the _decelerate_ method. Feel free to message me if you'd like, I can be more specific.